scripts/common/kubernetes.sh

```diff
@@ -289,6 +289,7 @@ function spinner_containerd_is_healthy() {
 # With AWS NLB kubectl commands may fail to connect to the Kubernetes API immediately after a single
 # successful health check
 function spinner_kubernetes_api_stable() {
+    echo "Waiting for kubernetes api health to report ok"
```

Comment:
generally we do this in the calling function, not the spinner code itself, but this'll be fine